openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModel.java

```diff
@@ -109,7 +109,7 @@ public void close() throws Exception {
      * @return the feature contributions.
      * @since 1.2.2
      */
-    public double[] getFeatureContributions(final Instance instance) {
+    protected double[] getFeatureContributions(final Instance instance) {
```

Comment:
is this a breaking change?

Reply:
no, it is not, because I just added this method, and 1.2.2 is not released yet.

Reply:
Can i close this @antssilva96 ?

Reply:
yes! Thanks Sheng